torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -819,6 +829,17 @@ def coordinate_descent_tuning(self, launcher, *args, **kwargs):
 
         config2launcher = {launcher.config: launcher}
 
+        # TODO: should we just load the kernels ahead of time if we know we're going to call this?
+        if self.fn.fn is None:
```

Comment:
@jansel Here's a second place where we may call _precompile_config on the parent. Specifically, when coordesc tuning is turned on, we still need self.fn in the parent process because we run precompile_config(). 

Interestingly enough, this almost always cache hits so that we *don't* need to reload the kernel, because we would have just finished compiling it in the worker; but someone can be running with `force_disable_caches`, etc, so we can't assume we will always cache hit.

Reply:
It's possible that it may be worth keeping a version of async_compile.triton that loads all the kernels in the parent process before sending it over the wire for cases like this, which could improve performance for cases where this function is turned on. But that's a lot of extra complexity in the code, so not sure if it's worth